frontend/src/Navbar.tsx

```diff
@@ -66,14 +60,37 @@ const Navbar: FunctionComponent = () => {
           >
             <Hamburger />
           </button>
+          {mobileMenu && (
```

Comment:
This was moved from below (out of the section that was guarded by user session `currentSession.isLoggedIn`). The margin was increased to `mt-8` to account for the navbar height.